build.gradle

```diff
@@ -36,10 +36,8 @@ dependencies {
 }
 
 processResources {
-	inputs.property 'version', version
-
```

Comment:
These lines are needed as the task wont be updated if the version changes.

Reply:
Ahh... makes sense